install/helm/agones/templates/service/allocation.yaml

```diff
@@ -276,3 +276,23 @@ data:
 {{- else }}
   tls-ca.crt: {{ .Files.Get "certs/allocator/server.crt" | b64enc }}
 {{- end }}
+
+---
+# Default allocation client secret
+{{- if .Values.agones.allocator.generateClientTLS }}
+{{- $selfSignedCert := genSelfSignedCert "allocator-client.default" nil nil 3650 }}
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/tls
+metadata:
+  name: allocator-client.default
+  namespace: {{ .Release.Namespace }}
```

Comment:
This needs to be stored in the gameserver namespace.

Reply:
https://github.com/googleforgames/agones/blob/master/install/helm/agones/templates/serviceaccounts/sdk.yaml#L20